src/role-access/containers/allocate-role/allocate-role-home/allocate-role-home.component.ts

```diff
@@ -95,6 +96,7 @@ export class AllocateRoleHomeComponent implements OnInit, OnDestroy {
     if (this.route.snapshot.queryParams) {
       this.caseId = this.route.snapshot.queryParams.caseId ? this.route.snapshot.queryParams.caseId : null;
       this.jurisdiction = this.route.snapshot.queryParams.jurisdiction ? this.route.snapshot.queryParams.jurisdiction : null;
+      this.caseType = this.route.snapshot.queryParams.caseType ? this.route.snapshot.queryParams.caseType : null;
```

Comment:
there are lots of repeated lines in this block`this.route.snapshot.queryParams` destructuring could be used to avoid this and improve readability e.g:

`const { caseId, caseType, ... } = this.route.snapshot.queryParams`

`this.caseType = caseType ? caseType : null;`

Reply:
Agree, good point will do it